package.json

```diff
@@ -30,13 +30,13 @@
     "/index.css"
   ],
   "dependencies": {
-    "lit-html": "^1.1.1"
+    "lit-html": "^1.2.1"
   },
   "devDependencies": {
     "@netflix/element-server": "^1.0.12",
     "@netflix/x-test": "^1.0.0-rc.11",
-    "eslint": "^6.0.1",
-    "puppeteer": "^1.11.0",
-    "tap-parser": "^9.0.0"
+    "eslint": "^7.4.0",
+    "puppeteer": "^5.0.0",
```

Comment:
☝️ these folks have been pumping out some major releases lately. I feel like it was 1.x.x not too long ago 😅